### ISSUE TITLE
Request: Hide columns, such as "added by"

### ISSUE THREAD

**Issue by xtbqimuxmvtx** [Wingysam/Christmas-Community#154]
Reason for adding issue is some funky rendering causing names to overflow to next line, I don't even want the Added by field, and it's ugly with text wrapping.

For example this entry added by my brother-in-law Joshua.

![image](https://github.com/user-attachments/assets/b2199c28-8a44-4022-860b-ea03154efc3a)

Wraps to `Joshu + a` at every resolution I can try that uses the inbuilt horizonal table view

**Comment by JadElClemens** [comment]
Agreed that this field is often not useful. "Pledged by" is more often useful, I reckon 90% of the time the pledger of an item not added by the list owner is the one that suggested the item. And 95% of the time in my family, items are bought from those added by the list owner.

If we want to preserve this information, I might suggest an information (circled "i") hoverable element that shows a popup with ex. "Added by: Adam" and optionally "Pledged By: Bob"
